### PR TITLE
[test] Allow set request/response size in interop soak test

### PR DESCRIPTION
**Pull request by mohanli-ml**
Internal bug: b/289109827

### CODE REVIEW

File: test/cpp/interop/client.cc [hunk 1]
```diff
@@ -120,6 +120,9 @@ ABSL_FLAG(int32_t, soak_min_time_ms_between_rpcs, 0,
 ABSL_FLAG(int32_t, iteration_interval, 10,
           "The interval in seconds between rpcs. This is used by "
           "long_connection test");
+ABSL_FLAG(int32_t, soak_request_size, 271828, "The request size in a soak RPC");
```
Comment (apolcyn) [inline test/cpp/interop/client.cc:123]: nit: let's mention in the comments that by default, sizes are configured as per the "large unary" spec

File: test/cpp/interop/interop_client.h [hunk 2]
```diff
@@ -146,21 +151,26 @@ class InteropClient {
...
+                       const int32_t overall_timeout_seconds,
+                       const int32_t request_size, const int32_t response_size);
 
   ServiceStub serviceStub_;
   /// If true, abort() is not called for transient failures
   bool do_not_abort_on_transient_failures_;
   // Load Orca metrics captured by the custom LB policy.
   LoadReportTracker load_report_tracker_;
+
+  static constexpr int kLargeRequestSize = 271828;
```
Comment (apolcyn) [inline test/cpp/interop/interop_client.h]: Do these constants redefined here? If not can we leave them as they were next to the other constants?

Comment (mohanli-ml) [reply test/cpp/interop/interop_client.h]: Moved it back. PTAL.

File: test/cpp/interop/client.cc (hunk 1 above)
Comment (mohanli-ml) [reply test/cpp/interop/client.cc]: Done.

**Review by apolcyn — APPROVED** [review]

**Pull request merged**